src/main/java/bio/terra/cli/command/app/Enable.java

```diff
@@ -15,17 +14,17 @@
   @CommandLine.Parameters(
       index = "0",
       description = "name of the application to enable: ${COMPLETION-CANDIDATES}")
-  private SupportedApp app;
+  private bio.terra.cli.apps.interfaces.Enable.EnableApp app;
 
   @Override
   public Integer call() {
-    if (app.enableStopPattern) {
-      GlobalContext globalContext = GlobalContext.readFromFile();
-      WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+    GlobalContext globalContext = GlobalContext.readFromFile();
+    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
 
-      new AuthenticationManager(globalContext, workspaceContext).loginTerraUser();
-      app.getToolHelper(globalContext, workspaceContext).enable();
-    }
+    new AuthenticationManager(globalContext, workspaceContext).loginTerraUser();
+    bio.terra.cli.apps.interfaces.Enable appHelper = app.getAppHelper();
+    appHelper.setContext(globalContext, workspaceContext);
+    appHelper.enable();
```

Comment:
I think we don't have enough examples to make good decisions here. We don't even know whether this enable/stop pattern will carry over to other apps. So, I'd like to just postpone this conversation until we have more firm constraints. Thank you for the suggestions though, I'll keep them in mind if we do decide to move forward with this pattern.

Regarding the value of listing supported apps:
There are some "apps"=client-side tools that I think we'd like to call in a pass-through manner. For example:
`terra nextflow run hello`
`terra gcloud ls`
The goal is to have the commands look exactly as they would if you ran them on your local machine, except with a `terra` prefix. The purpose of the prefix is to run the command in the context of a workspace. Currently, this context just means running with the credentials and google project set for you already (see terra_init.sh script). I think it will also soon mean that data references scoped to the workspace are resolved for you already (whatever that means -- env vars, flat file, something more fancy).

These pass-through apps are not visible in the top-level help (that was confusing because they don't really fit into the other command groupings we have there -- e.g. notebooks, workspace, auth). So `terra app list` is a way to see the ones we support.